src/s11/main.c

```diff
@@ -48,6 +49,7 @@ struct thread_pool *g_tpool_tipc_reader_s11;
 extern char processName[255];
 extern int pid;
 
+#define S11_IPC_MSG_BUF_LEN 1024
```

Comment:
Not right place to define this. Please open a cleanup issue where we need to manage all include files.

Reply:
Created jira issue.